source/controllers/representing-multiple-models.md

```diff
@@ -24,10 +24,7 @@ each song:
 </ul>
 ```
 
-You can use the controller to collect aggregate information about
-the model it hosts. For example, imagine we want to display the
-number of songs that are over 30 seconds long. We can add a new computed
-property called `longSongCount` to the controller:
+You can use the controller to collect aggregate information about its `model`. Imagine we want to display the number of songs that are over 30 seconds long. We can add a new computed property called `longSongCount` to the controller:
```

Comment:
You forgot to manually introduce line breaks like the rest of the guides, can you add them?